### PR TITLE
Add Tanzania national and religious holiday

### PR DESCRIPTION
**Pull request by alphaolomi**
A modified PR by @omakei, fixing issues

- #86

### CODE REVIEW

**Comment by alphaolomi** [comment]
@Nielsvanpach 👀

File: src/Countries/Tanzania.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+use IntlCalendar;
+
+class Tanzania extends Country
+{
+    public function countryCode(): string
+    {
+        return 'tz';
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
```
Comment (Nielsvanpach) [inline src/Countries/Tanzania.php:15]:
```suggestion
```

File: src/Countries/Tanzania.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            'Farmers Day (Nane Nane Day)' => '08-08',
+            'Christmas Day' => '12-25',
+            'Boxing Day' => '12-26'
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
```
Comment (Nielsvanpach) [inline src/Countries/Tanzania.php:31]:
```suggestion
        $easter = $this->easter($year);
```

File: src/Countries/Tanzania.php [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            ->setTimezone('Africa/Dar_es_Salaam');
+
+        // Eid al-Fitr is on the first day of the 10th month in the calendar
```
Comment (Nielsvanpach) [inline src/Countries/Tanzania.php:34]: I'm still a bit hesitant to merge PRs with Islamic holidays, as there still isn't a good implementation yet.

Comment (omakei) [reply src/Countries/Tanzania.php:34]: Let me remove Islamic holidays until good implementation  is found.

**Comment by alphaolomi** [comment]
@omakei @Nielsvanpach revised as advised.

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**